pretests/log_versions/log_versions.py

```diff
@@ -0,0 +1,50 @@
+r"""
+Summary
+-------
+
+Preserve docker versions in state files.
+
+Operational Summary
+-------------------
+
+#. Run 'docker version'; use DockerVersion module to parse output
+#. Run 'rpm -q docker|docker-latest' (whichever one is in use)
```

Comment:
My immediate goal is to have my jenkins-analyze script cross-check a list of known failures in known docker/docker-latest builds. For that, it needs to check the rpm currently in use. Fetching `docker_rpm` is an easy way to do that.

Logging all rpm versions (docker, docker-latest, runc, plugins, selinux) is preferable but that wasn't my immediate goal; and it actually blocks my immediate goal because I then need to devise a mechanism to log whether docker or docker-latest is in use. Any suggestion on how to solve that?